tests/angular/custom-ignore-attributes/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+const {ignoreAttributes} = require('@ni/eslint-config-angular/options');
+
+module.exports = {
+    overrides: [{
+        files: ['*.html'],
+        rules: {
+            '@angular-eslint/template/i18n': [
+                'error',
+                {
+                    checkId: false,
+                    ignoreAttributes: [...ignoreAttributes.all, 'custom-field']
+                }
+            ],
+        }
+    }, {
+        files: ['*.spec.ts*.html'],
+        rules: {
+            '@angular-eslint/template/i18n': 'off'
+        }
+    }]
```

Comment:
If an app wants to add additional custom attributes to ignore i18n this is what it looks like. It's kinda annoying to have to re-disable the checks in tests but maybe that's fine? :/

@jattasNI @rbell517  thoughts on if we should document this pattern for ignoring project-specific attributes? I'm not really sure how often it'll come-up. Should we pre-emptively run local builds of the lints in more projects to see if many apps will need overrides? If they do should we try and come up with better ways to configure this?

Reply:
I personally don't think we need to document anything for this. The tests themselves are reasonable documentation and realistically its most likely that if there are extra attributes to ignore they are going to come from a library and we would want to add to our ignore list instead.

Reply:
I'm ok to wait until someone asks how to do it.